source/content/partials/plans.md

```diff
@@ -21,9 +21,9 @@ The List Price is the set price for new sites created after November 15, 2018 th
 
```

Comment:
@carly-armstrong are you also updating the info on https://pantheon.io/plans/agency-preferred-pricing ?

Reply:
Uncerrtain if that page pulls from this link, but  I believe the web team will be updating it at 8am PST.